### PR TITLE
Feat/continue software arch

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | internal !23140

### CODE REVIEW

**Comment by orthagh** [comment]
Yes, the order requires a filter feature.
I will to reuse the one present in Historical tab

**Comment by orthagh** [comment]
image


Ready for review

**Review by trasher — APPROVED** [review]

**Review by g-bougard — APPROVED** [review]

File: js/log_filters.js [hunk 1]
```diff
@@ -56,7 +56,13 @@ $(function() {
             }
         });
 
-        $('.log_history_filter_row [name^="filters\\["]').on('input', handleFilterChange);
+        var delay_timer = null;
+        $('.log_history_filter_row [name^="filters\\["]').on('input', function() {
+            clearTimeout(delay_timer);
+            delay_timer = setTimeout(function() {
+                handleFilterChange();
+            }, 800);
```
Comment (cedric-anne) [inline js/log_filters.js:64]: 800ms seems a bit long to me, users may think it is lagging.
```suggestion
            }, 500);
```

**Review by cedric-anne — APPROVED** [review]

File: js/log_filters.js (hunk 1 above)
Comment (orthagh) [reply js/log_filters.js:64]: i tested originally with 500ms, it seemed a little too short

**Pull request merged**